tee-worker/client-api/parachain-api/prepare-build/interfaces/sidechain/definitions.ts

```diff
@@ -0,0 +1,57 @@
+export default {
```

Comment:
can this be moved to `sidechain-api` ? next two files too?

Reply:
I believe it won't be a **minor** issue anymore because it will also involve changes to the `ts-tests`. Considering that this PR already involves two PRs and a todo, I suggest creating a new PR to address this.
:p

Reply:
The question is, which RPC API are we using these types for?
If they're used via the parachain RPC, the defs need to stay _here_; sucky duplication notwithstanding. TL;DR the RPC types are not (and will not be) included with the chain metadata; some history can be found [here](https://github.com/paritytech/substrate/issues/11181) (warning: rabbit hole :rabbit: :hole: :warning:).
If they're not being used at all tho, we can just yeet them :D